AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin.swift

```diff
@@ -18,7 +18,11 @@ final public class AWSDataStorePlugin: DataStoreCategoryPlugin {
     /// The Publisher that sends mutation events to subscribers
     var dataStorePublisher: DataStoreSubscribeBehavior?
 
-    let modelRegistration: AmplifyModelRegistration
+    /// The schema that holds a reference to its version and all registered models
+    let schema: DataStoreSchema
+
+    /// The DataStore configuration
+    let configuration: DataStoreConfiguration
```

Comment:
On line72, in the function `configure(using configuration: Any)`, we pass in the amplify configuration (I think), so we might want to rename this to something like dataStoreConfiguration